spec/models/scheme_spec.rb

```diff
@@ -209,4 +209,16 @@
       end
     end
   end
+
```

Comment:
@kosiakkatrina I've added this test in response to you comment - thanks for the suggested solution!